doc-content/enterprise-only/project-data/preview-editor-delete-test-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='preview-editor-delete-test-proc']
+= Deleting a test scenario
+
+You can delete existing test scenarios which were created using the *Test Scenario (Preview)* editor.
+
+.Procedure
+. From the *Test Scenario (Preview)* editor toolbar on the upper-right, click *Delete*.
+. In the *Confirm Delete* window,
+* If you wish to add a comment regarding the deletion of the test scenario, click *add a comment*.
```

Comment:
Suggestion:
To add a comment regarding the deletion of the test scenario, click *add a comment*.

Reply:
Changed as suggested.